torchgeo/datasets/xview.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""xView2 dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, List, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytorch_lightning as pl
+import torch
+from PIL import Image
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets.utils import dataset_split
+from .geo import VisionDataset
+
+
+class xView2(VisionDataset):
```

Comment:
Classes should use CamelCase: https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Changed everywhere to XView2